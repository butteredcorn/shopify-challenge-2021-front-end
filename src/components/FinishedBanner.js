import styled from 'styled-components'
import '../styles/components/FinishedBanner.css'

const BannerContainer = styled.div`
    background-color: ${props => props.theme.bg.primary};
    color: ${props => props.theme.text.primary};
    border: 1px solid ${props => props.theme.bg.primary};
`

const Banner = styled.div`
    background-color: ${props => props.theme.bg.secondary};
    color: ${props => props.theme.text.tertiary};
    border: 1px solid ${props => props.theme.bg.primary};
`

const FinishedBanner = () => {
    return (
        <BannerContainer className="banner-container">
            <Banner>
                <h2>Thank you for submitting your nominations for Shoppies!</h2>
            </Banner>
        </BannerContainer>
    );
  };
export default FinishedBanner;