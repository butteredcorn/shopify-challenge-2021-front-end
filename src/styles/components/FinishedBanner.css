.banner-container {
    max-width: 1200px; 
    min-height: 10vh;
}

.banner-container > * {
    margin: 1rem;
    padding: 1rem;
    font-size: 1.5rem;
    text-align: center;
}