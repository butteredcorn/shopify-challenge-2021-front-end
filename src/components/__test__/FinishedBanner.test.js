import { render, screen, cleanup } from '@testing-library/react';
import ThemesProvider from '../../providers/ThemeProvider'
import renderer from 'react-test-renderer'
import FinishedBanner from '../FinishedBanner';

afterEach(cleanup);

it('renders FinishedBanner component with message "Thank you for submitting your nominations for Shoppies!"', () => {
  render(<ThemesProvider><FinishedBanner/></ThemesProvider>);
  const message = screen.getByText(/Thank you for submitting your nominations for Shoppies!/i);
  expect(message).toBeInTheDocument();
});

it('matches a snapshot', () => {
  const json = renderer.create(<ThemesProvider><FinishedBanner/></ThemesProvider>).toJSON();
  expect(json).toMatchSnapshot();
});