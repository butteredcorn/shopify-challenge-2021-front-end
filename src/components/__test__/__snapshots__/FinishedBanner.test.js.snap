// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches a snapshot 1`] = `
<div
  className="sc-bdvvaa isJxqY banner-container"
>
  <div
    className="sc-gsDJrp bphuEr"
  >
    <h2>
      Thank you for submitting your nominations for Shoppies!
    </h2>
  </div>
</div>
`;
